src/app/hooks/useViewTracker/index.tsx

```diff
@@ -91,26 +98,20 @@ const getComponentViewTracker = (eventTrackingData?: EventTrackingData) => {
         ].every(Boolean);
 
         if (shouldSendEvent) {
-          if (optimizely) {
-            const eventName = OPTIMIZELY_CONFIG.viewClickAttributeId;
-
+          if (sendOptimizelyEvents && optimizelyVariation) {
             const overrideAttributes = {
-              ...optimizely.user.attributes,
-              [`viewed_${eventName}`]: true,
+              ...optimizely?.user.attributes,
             };
 
-            optimizely.track(
+            optimizely?.track(
               optimizelyMetricNameOverride
                 ? `${optimizelyMetricNameOverride}_views`
-                : 'component_views',
-              optimizely.user.id as string,
+                : 'component-views',
+              optimizely?.user.id as string,
               overrideAttributes,
             );
           }
 
-          const optimizelyVariation =
-            optimizely?.getVariation(OPTIMIZELY_CONFIG.ruleKey) || null;
```

Comment:
Calling this function from the OptimizelyContext always returned a variation, even if the service was included in our `enabledExperimentsList.js` array. So we can't use this function for getting the MVT variation.